admin/orders.php

```diff
@@ -1022,6 +1036,11 @@ function couponpopupWindow(url) {
         if (zen_not_null($oInfo->last_modified)) $contents[] = array('text' => TEXT_DATE_ORDER_LAST_MODIFIED . ' ' . zen_date_short($oInfo->last_modified));
         $contents[] = array('text' => '<br />' . TEXT_INFO_PAYMENT_METHOD . ' '  . $oInfo->payment_method);
         $contents[] = array('text' => '<br />' . ENTRY_SHIPPING . ' '  . $oInfo->shipping_method);
+        $orderLanguageQuery = $db->Execute("SELECT name
+                                            FROM " . TABLE_LANGUAGES . "
+                                            WHERE languages_id = " . $oInfo->language_id);
+        $orderLanguage = $orderLanguageQuery->fields['name'];
+        $contents[] = array('text' => '<br />' . TEXT_INFO_ORDER_LANGUAGE . ' ' . $orderLanguage);
```

Comment:
Similar to the earlier note, this could be optimized to avoid a query by:
```php
$orderLanguage = $lng->get_language_data_by_id($oInfo->language_id);
$contents[] = array('text' => '<br />' . TEXT_INFO_ORDER_LANGUAGE . ' ' . $orderLanguage['name']);
```